build.sbt

```diff
@@ -39,13 +35,11 @@ lazy val microservice = Project(appName, file("."))
         play26 = compilePlay26 ++ testCompilePlay26
     )
   )
+  .settings(PlayCrossCompilation.playCrossCompilationSettings)
   .settings(
-    resolvers := Seq(
-      Resolver.jcenterRepo,
-      Resolver.bintrayRepo("hmrc", "releases")
-    )
+    // TODO Use the project/ScoverageSettings.scala as soon as you upgraded to SBT 1.5
```

Comment:
WARNING
There exists JIRA ticket DC-4038 demanding for SBT to be upgraded to version 1.5